### ISSUE TITLE
wget not found on Synology NAS

### ISSUE THREAD

**Issue by epheterson** [DocDrydenn/kiwix-zim#1]
Thanks so much for making this! I've been looking for something like that since first setting up Kiwix ages ago.

One small issue is that it's not discovering that `wget` is installed on Synology NAS, although it is there and working fine. Besides that, there are no errors in the script and it seems to update the files.

```
2. Checking Required Packages...

dpkg-query: no packages found matching wget
   ✗ wget: Not Found

2a. Installing Missing Packages:

./kiwix-zim.sh: line 82: apt: command not found
```

wget is there and works fine:
```
$ which wget
/bin/wget
```

**Comment by DocDrydenn** [comment]
I don't have a Synology to play around in, but I'm guessing they don't use `dpkg-query` or `apt`.

The commands used to check for and install required packages/tools are: 

`dpkg-query -W --showformat='${Status}\n' $REQUIRED_PKG|grep "install ok installed"`
&
`apt install -y $install_pkgs`

(in this case `$REQUIRED_PKG` and `$install_pkgs` are just strings that equal "wget")

I do not know where I'd even begin to try to fix this (and not break it for regular Linux systems). Maybe I can find a more universal way of checking for needed packages/tools...

In the mean time, did it continue on and work properly or did it just crash out?

**Comment by DocDrydenn** [comment]
Can't get any more simple than this. LOL

Try running `command -v wget` on your Synology...

If `wget` is found, it should output the path. If not found, there'll be no output at all.

**Comment by epheterson** [comment]
Yeah `command -v wget` works, seems to be essentially the same as `which wget`

```
$ command -v wget
/bin/wget
```

**Comment by DocDrydenn** [comment]
Nice. Change committed to main.
